src/Components/Settings/Settings.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../style/variables.scss';
+@import '../../style/mixins.scss';
+
+.settingsContainer {
+    display: flex;
+    flex-direction: column;
+    width: 30%;
+    margin: auto;
+    background: $darkBlue;
+    color: #F8F8FF;
+    border-radius: 0.7em;
+    padding: 15px;
+}
+
+.settingsHeader {
+    text-align: center;
+    margin: 0;
+}
+
+.intervalContainer {
+    display: flex;
+    flex-direction: row;
```

Comment:
I think that flex containers are inherently `rows` so we could trim this line ✂️